Gemfile.lock

```diff
@@ -0,0 +1,158 @@
+GIT
```

Comment:
It is the only way to ensure consistent setups. As said, there is a point to be made about libraries development, but imo checking in the lock file is always the better approach.